packages/boxel/addon/components/boxel/action-chin/usage.gts

```diff
@@ -235,6 +236,13 @@ export default class ActionChinUsage extends Component<Signature> {
         @value={{this.stepNumber}}
         @onInput={{fn (mut this.stepNumber)}}
       />
+      <Args.Bool
+        @name="hideLock"
+        @description="When true, suppress the 'Actions only visible to you' message and lock icon."
+        @defaultValue={{false}}
+        @value={{this.hideLock}}
+        @onInput={{fn (mut this.hideLock)}}
+      />
```

Comment:
This needs to be consumed in the invocation 🤓

Reply:
In the [preview](https://boxel.stack.cards/boxel-tweaks/#/docs?s=Components&ss=%3CBoxel%3A%3AActionChin%3E) checking the box does nothing:

<img width="1175" alt="Boxel Components 2022-10-12 14-54-54" src="https://user-images.githubusercontent.com/43280/195435476-15bb01ad-764f-4765-8885-edcd9208e628.png">